File: src/main/java/dev/taboo/taboo/util/ParseBytes.java
package dev.taboo.taboo.util;

public class ParseBytes {

    public static String parseBytes(double bytes) {
        String converted;
        double kb = bytes / 1024;
        double mb = kb / 1024;
        double gb = mb / 1024;
        if (gb > 0) {
            converted = gb + " GB";
        } else if (mb > 0) {
            converted = mb + " MB";
        } else {
            converted = kb + " KB";
        }
        return converted;
    }

}

File: src/main/kotlin/dev/taboo/taboo/util/ParseBytes.kt
package dev.taboo.taboo.util

object ParseBytes {

    @JvmStatic
    fun parseBytes(bytes: Double): String {
        val converted: String
        val kb = bytes / 1024
        val mb = kb / 1024
        val gb = mb / 1024
        converted = if (gb > 0) {
            "$gb GB"
        } else if (mb > 0) {
            "$mb MB"
        } else {
            "$kb KB"
        }
        return converted
    }

}